site/content/docs/5.3/components/dropdowns.md

```diff
@@ -693,33 +693,67 @@ Add `.dropdown-menu-end` to a `.dropdown-menu` to right align the dropdown menu.
 
 If you want to use responsive alignment, disable dynamic positioning by adding the `data-bs-display="static"` attribute and use the responsive variation classes.
 
+In the case of using the split button style, dynamic positioning should not be disabled, otherwise the menu will not be accurately positioned to the split button.
+
 To align **right** the dropdown menu with the given breakpoint or larger, add `.dropdown-menu{-sm|-md|-lg|-xl|-xxl}-end`.
 
 {{< example >}}
-<div class="btn-group">
-  <button type="button" class="btn btn-secondary dropdown-toggle" data-bs-toggle="dropdown" data-bs-display="static" aria-expanded="false">
-    Left-aligned but right aligned when large screen
-  </button>
-  <ul class="dropdown-menu dropdown-menu-lg-end">
-    <li><button class="dropdown-item" type="button">Action</button></li>
-    <li><button class="dropdown-item" type="button">Another action</button></li>
-    <li><button class="dropdown-item" type="button">Something else here</button></li>
-  </ul>
+<div class="d-inline-grid gap-4">
```

Comment:
These classes could go on the example tag, but also we should probably split these across separate `example` tags.

Reply:
Just made a commit taking these changes into account: https://github.com/twbs/bootstrap/pull/38988/commits/c2e2f1acad9e1d5a54ba0f2d916650d2ac95aff4. IMO, I'd stick to have the examples together since it's done this way in the page.